EAD/EADSession4/WebContent/registration.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+	<table>
+		<tr>
+			<td><img src="Images/metacube.jpg" height="20%"></td>
+			<td align="right" width="70%"></td>
+		</tr>
+	</table>
+	<hr />
+	<h1 align="center">Employee Details</h1>
+	<form align="center" action="RegistrationServlet" method="post">
+		<table align="center" cellspacing="10px" frame="box">
+			<tr>
+				<td width="100px" align="Left">First Name:</td>
+				<td><input type="text" name="firstName"
+					placeholder="Enter First Name" required />
+				</td>
+			</tr>
+			<tr>
+				<td width="100px" align="Left">Last Name:</td>
+				<td><input type="text" name="lastName"
+					placeholder="Enter Last Name" required />
+				</td>
+			</tr>
+			<tr>
+				<td width="100px" align="Left">Date of birth:</td>
+				<td align="Left"><input type="Date" name="dateOfBirth"
```

Comment:
either we should use datepicker or mention which type of date format is expected (Wrong date is getting saved in database)